src/index.html

```diff
@@ -2,31 +2,64 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
+
     <meta
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     />
+
     <meta
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+
     <title>Document</title>
+
     <link
       rel="stylesheet"
-      href="style.css"
+      href="./style.css"
     />
   </head>
+
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <main class="main">
+      <div class="main__form">
+        <form
+          class="search-form search-form--large"
+          data-qa="big"
+        >
+          <span
+            href="#"
+            class="search-form__search-icon search-form__search-icon--large"
+          ></span>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+          <input
+            type="text"
+            data-qa="keypress"
+            placeholder="Try “Los Angeles“"
+            class="search-form__input search-form__input--large"
+          />
```

Comment:
The placeholder text 'Try “Los Angeles“' should not include quotes as they are unnecessary in this context and can be confusing for users. Consider simplifying it to just 'Try Los Angeles'.